sdk/servicebus/service-bus/samples/v7/javascript/package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "azure-service-bus-samples-js",
+  "name": "@azure-samples/service-bus-js",
```

Comment:
was there an error for the old name `azure-service-bus-samples-js`?

Reply:
not sure, it was a part of the main branch when I ran git checkout

Reply:
This is generated though, must be a change from dev-tool

Reply:
yeah we just weren't sure why this change appears in the PR, as main branch has `@azure-samples/service-bus-js` already